packages/@expo/cli/src/start/doctor/dependencies/__tests__/ensureDependenciesAsync-test.ts

```diff
@@ -15,12 +15,23 @@ describe(createInstallCommand, () => {
   it(`formats npm`, () => {
     expect(
       createInstallCommand({
-        isYarn: false,
+        manager: 'npm',
         packages: [
           { pkg: '@other/pkg', file: '' },
           { pkg: 'bacon', file: '', version: '~1.0.0' },
         ],
       })
     ).toBe('npm install @other/pkg bacon@~1.0.0');
   });
+  it(`formats pnpm`, () => {
+    expect(
+      createInstallCommand({
+        manager: 'pnpm',
```

Comment:
![thank-you](https://user-images.githubusercontent.com/1203991/168051720-269dfd6d-e5ea-4515-aef0-fc67a1a3e909.gif)